gbpservice/nfp/configurator/lib/demuxer.py

```diff
@@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
         """
 
         sa_info_list = []
+        vendor_map = {const.FIREWALL: const.VYOS,
+                      const.LOADBALANCER: const.HAPROXY}
+
+        service_vendor = request_data['info']['service_vendor']
+        if str(service_vendor) == 'None':
```

Comment:
Config_orchestrator doesn't send us service_vendor info for *aaS APIs.

Regards,
Deepak

On Wed, Apr 27, 2016 at 2:07 PM, njagadish <notifications@github.com> wrote:

> In gbpservice/nfp/configurator/lib/demuxer.py
> <https://github.com/oneconvergence/group-based-policy/pull/251#discussion_r61222033>
> :
>
> > @@ -105,45 +103,56 @@ def get_service_agent_info(self, operation, service_type, request_data):
> >          """
> >
> >          sa_info_list = []
> > +        vendor_map = {const.FIREWALL: const.VYOS,
> > +                      const.LOADBALANCER: const.HAPROXY}
> > +
> > +        service_vendor = request_data['info']['service_vendor']
> > +        if str(service_vendor) == 'None':
>
> Are we not mandating service_vendor? What are the cases where we get None
> for service vendor?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/oneconvergence/group-based-policy/pull/251/files/9c68054f9caa237f1086b0c64846ae0c6080839c#r61222033>
>